### PR TITLE
product cart added

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+<body>
+    <!-- nav-section -->
+    <nav class="navbar-content">
+        <h1 class="nav-title">WALDO Store</h1>
+        <div class="nav-container-search">
+            <select>
+                <option value="">All Category</option>
+                <option value="">Fuel</option>
+                <option value="">Engines</option>
+                <option value="">Fins</option>
```
Comment (pratik-kumar-sahu) [inline pages/cart.html:23]: - in deployed link this texts are not visible.

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
+.cart-main-section{
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    gap: 2rem;
+    margin-top: 8rem;
+    margin-left: 3rem;
+    margin-bottom: 3rem;
+    justify-content: space-between;
```
Comment (Mohasin-Haque) [inline styles/cart.css]: you can use shorthand property for this

File: styles/cart.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+}
+.remove-all:hover{
+    background: var(--primary-clr-hover);
+    color: var(--primary-clr);
+}
+
+
+/* horizontal-card */
+.cart-card-container{
+    display: flex;
```
Comment (Mohasin-Haque) [inline styles/cart.css:41]: make utility class for flex

File: styles/cart.css [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    display: inline-block;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+}
+.remove-btn, .later-btn{
+    margin: 2rem 2rem 1rem;
+    padding: 0.3rem 0.8rem;
+    border: none;
+    border-radius: 10px;
```
Comment (Mohasin-Haque) [inline styles/cart.css:70]: Try using relative units

**Review by pratik-kumar-sahu — APPROVED** [review]
- Code looks great. You can use a code formatter to get rid of uneven spaces and alignments.
- Nice work go ahead with merging.

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good to me. But needs to work on the comments

**Pull request merged**